optuna/visualization/matplotlib/_utils.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+from optuna.distributions import LogUniformDistribution
+from optuna.trial import FrozenTrial
+from optuna.visualization.matplotlib import _matplotlib_imports
+
+__all__ = ["is_available"]
+
+
+def is_available() -> bool:
+    """Returns whether visualization is available or not.
+
+    .. note::
+
+        :mod:`~optuna.visualization` module depends on plotly version 4.0.0 or higher. If a
+        supported version of plotly isn't installed in your environment, this function will return
+        :obj:`False`. In such case, please execute ``$ pip install -U plotly>=4.0.0`` to install
+        plotly.
+
+    Returns:
+        :obj:`True` if visualization is available, :obj:`False` otherwise.
```

Comment:
Could you update the docstring to match the `matplotlib`?
```suggestion
    """Returns whether visualization with `matplotlib` is available or not.

    .. note::

        :mod:`~optuna.visualization` module depends on matplotlib version 3.0.0 or higher. If a
        supported version of matplotlib isn't installed in your environment, this function will return
        :obj:`False`. In such a case, please execute ``$ pip install -U matplotlib>=3.0.0`` to install
        matplotlib.

    Returns:
        :obj:`True` if visualization with `matplotlib` is available, :obj:`False` otherwise.
```